grizli/prep.py

```diff
@@ -5595,3 +5808,19 @@ def extract_fits_log(file='idk106ckq_flt.fits', get_dq=True):
                 log['DQv{0}'.format(chip)] = list(dq[mask].astype(str))
 
     return log
+
+
+def check_isJWST(infile=''):
+    '''
+    Check if a file or list of files is JWST image. 
+    Parameters:
+    - infile: fits file
+    
+    Returns:
+    - True or False
+    '''
+
+    hdu = pyfits.open(infile)
+    isJWST = (hdu[0].header['TELESCOP'] == 'JWST')
```

Comment:
This is now breaking in the test.  First add a check to see if the 'TELESCOP' keyword exists, and I guess assume `isJWST=False` if it doesn't.  

Also please try to make the formatting of the docstring look like that of other functions, e.g., 
```python
def column_values_in_list(col, test_list):
    """Test if column elements "in" an iterable (e.g., a list of strings)

    Parameters
    ----------
    col : `astropy.table.Column` or other iterable
        Group of entries to test

    test_list : iterable
        List of values to search

    Returns
    -------
    test : bool array
        Simple test:
            >>> [c_i in test_list for c_i in col]
    """
    test = np.array([c_i in test_list for c_i in col])
    return test
```

Reply:
Fixed both here (sorry!): 91838945f5bf1813894b49f9807a84bfb0e3118f